StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/PaneLayoutView.swift

```diff
@@ -62,11 +68,7 @@ final class PaneLayoutView {
     }
 
     /// Adds this `PaneLayoutView` to the provided view.
-    /// In order to keep the footer view above the keyboard;
-    /// - Set `keepFooterAboveKeyboard: true`.
-    /// - Hold onto this instance of `PaneLayoutView` on the view controller presenting it.
-    /// This is required to prevent the keyboard observer notifications be removed.
-    func addTo(view: UIView, keepFooterAboveKeyboard: Bool = false) {
+    func addTo(view: UIView) {
```

Comment:
Moved the `keepFooterAboveKeyboard` to the `init` since it seems to make more sense there.